File: HFY.Entities/Concrete/LastikTurler.cs
using HFY.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace HFY.Entities.Concrete
{
    public class LastikTurler : IEntity
    {
        [Key]
        public int LastikTurID { get; set; }

        public string Kod { get; set; }

        public string Ad { get; set; }

        public int Sira { get; set; }
        public int OlusturanId { get; set; }

        public DateTime OlusturmaTarihi { get; set; }

        public int DuzenleyenId { get; set; }

        public DateTime DuzenlemeTarihi { get; set; }

        public bool Aktif { get; set; }

        public bool ListeAktiflik { get; set; }
    }
}

File: HFY.MvcWebUI.Angular/ClientApp/src/app/lazy-general-page/classes/lastik-turler/lastik-turler.ts
export class LastikTurler {
    LastikTurID: number;
    Kod: string;
    Ad: string;
    Sira: number;
    OlusturanId: number;
    OlusturmaTarihi: Date | string;
    DuzenleyenId: number;
    DuzenlemeTarihi: Date | string;
    Aktif: boolean;
    ListeAktiflik: boolean;

    jqueryDataTableLastikTurler() {
        return [{ data: 'Sira' }, { data: 'Ad' }, { data: 'Kod' },
        { data: 'Aktif' } // İşlemler
        ]
    }
}
